COPYING

```diff
@@ -0,0 +1,502 @@
+                  GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
I think you can remvoe this whole file, and simply add a single like to point to the `COPYING` file in the recipe.